src/cmd/ksh93/tests/vi.exp

```diff
@@ -949,6 +949,14 @@ expect -re ".*\rKSH PROMPT:\[0-9\]+: $" { }
 send "\r"
 expect_prompt
 
+# Regrssion tests
+# https://github.com/att/ast/issues/682
+send "mkdir ~/local\r"
```

Comment:
Probably worth a comment that this depends on the *run_tests.sh* test runner setting `$HOME` to the temp test dir to avoid scribbling on the users home dir.